.chloggen/fix_sts_when_tls_is_enabled.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. tempostack, tempomonolithic, github action)
+component: tempostack
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note:  Allow insecure = false/true regardless of type of token used.
```

Comment:
Can we be more specific here? I am not sure what token does it concern

Reply:
Fixed